Tools/TestWebKitAPI/Tests/WebKitGtk/TestContextMenu.cpp

```diff
@@ -66,6 +66,7 @@ class ContextMenuTest: public WebViewTest {
     {
         g_assert_true(WEBKIT_IS_CONTEXT_MENU(contextMenu));
         test->assertObjectIsDeletedWhenTestFinishes(G_OBJECT(contextMenu));
+        g_assert_true(webkit_context_menu_get_event(contextMenu) == event);
```

Comment:
```suggestion
#if PLATFORM(GTK)
        g_assert_true(webkit_context_menu_get_event(contextMenu) == event);
#endif
```

Reply:
This shouldn't be needed because this test is GTK specific